dev/com.ibm.ws.anno_fat/bnd.bnd

```diff
@@ -12,23 +12,28 @@
 bVersion=1.0
 
 src: \
-	fat/src,\
-	test-applications/TestServlet40.war/src,\
-	test-applications/TestServlet40.jar/src
-	
-fat.project: true
-
-javac.source: 1.8
-javac.target: 1.8
+    fat/src, \
```

Comment:
I also saw that you checked in binary ear files in the anno_fat/test-applications/test-applications/fe directory structure.  Why are binary applications checked in?  Can they be removed?

Reply:
Also, I didn't see any updates to the anno_fat .classpath to add in the new test-applications to the list of source for the project.

Reply:
These, along with several under com.ibm.ws.anno, should be converted, but haven't yet.

Reply:
The .classpath file should be updated still to include these new source directories.  

So should the binary ones be removed?  Are they used for tests?  Do they have value?

Reply:
I will handle updating the .classpath in a separate PR after this PR is merged.

Reply:
The updating of the test-applications/fe applications is being tracked for future cleanup in #9225.